.github/workflows/promptflow-replay-test.yml

```diff
@@ -79,12 +79,31 @@ jobs:
       working-directory: ${{ env.testWorkingDirectory }}
       run: |
         gci env:* | sort-object name
-        $env:PYTHONPATH=""
         cp ${{ github.workspace }}/src/promptflow/dev-connections.json.example ${{ github.workspace }}/src/promptflow/connections.json
         python "../../scripts/building/run_coverage_tests.py" `
         -p promptflow `
         -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test `
         -l eastus `
         -m "unittest or e2etest" `
         --coverage-config ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test/.coveragerc
+    - name: Publish Test Results
+      uses: EnricoMi/publish-unit-test-result-action@v2
+      with:
+        check_name: "Replay Test [${{ steps.display_env.outputs.GITHUB_HEAD_REF }}](https://github.com/microsoft/promptflow/actions/workflows/promptflow-replay-test.yml?query=branch:${{ steps.display_env.outputs.GITHUB_HEAD_REF }}++)"
+        comment_title: "Replay Test [${{ steps.display_env.outputs.GITHUB_HEAD_REF }}](https://github.com/microsoft/promptflow/actions/workflows/promptflow-replay-test.yml?query=branch:${{ steps.display_env.outputs.GITHUB_HEAD_REF }}++)"
+        files: "**/test-results.xml"
+    - name: Generate Coverage Report
+      if: (success() || failure())
```

Comment:
is it possible to make this `always()`?

Reply:
They are the same. 
And success failure pair will exclude running coverage report if you have syntax error in run_coverage_tests.py.

Reply:
Besides, this is suggested ways generated by gh action-importer.